astro/src/content/docs/extend/code/lambdas/mfa-requirement.mdx

```diff
@@ -0,0 +1,315 @@
+    os;
+
+    /**
+     * @type {string}
+     */
+    userAgent;
+}
+
+class Location {
+    /**
+     * @type {string}
+     */
+    city;
+
+    /**
+     * @type {string}
+     */
+    country;
+
+    /**
+     * @type {double}
+     */
+    latitude;
+
+    /**
+     * @type {double}
+     */
+    longitude;
+
+    /**
+     * @type {string}
+     */
+    region;
+
+    /**
+     * @type {string}
+     */
+    zipcode;
+}
+
+class Policies {
+    /**
+     * This value will be set to the application's MFA login policy, if set.
+     * @type {string} - Possible values are [Disabled], [Enabled], or [Required].
+     */
+    applicationLoginPolicy;
+
+    /**
+     * If the Application has an MFA trust policy set, this value will be set to that policy.
+     * @type {string} - Possible values are [Any], [This], or [None].
+     */
+    applicationMultiFactorTrustPolicy;
+
+    /**
+     * This value will be set to the tenant's MFA login policy.
+     * @type {string} - Possible values are [Disabled], [Enabled], or [Required].
+     */
+    tenantLoginPolicy;
+}
+
+class StartInstant {
+    /**
+     * Every time /api/two-factor/login is completed for an application, the application Id and instant will be recorded here.
+     * see https://fusionauth.io/docs/reference/data-types#instants for instant documentation
+     * @type {Map<UUID, Instant>}
+     */
+    applications;
+
+    /**
+     * The first instant the trust was established for the user's tenant.
+     * see https://fusionauth.io/docs/reference/data-types#instants for instant documentation
+     * @type {Instant}
+     */
+    tenant;
+}
+
+class Trust {
+    /**
+     * The application that the MFA trust the user already has, was first associated with
+     * @type {string}
+     */
+    applicationId;
+
+    /**
+     * Additional attributes about the trust.
+     * @type {Map<String, Object>}
+     */
+    attributes;
+
+    /**
+     * When the MFA trust expires, see https://fusionauth.io/docs/reference/data-types#instants
+     * @type {Instant}
+     */
+    expirationInstant;
+
+    /**
+     * The twoFactorTrustId
+     * @type {String}
+     */
+    id;
+
+    /**
+     * When the MFA trust was created, see https://fusionauth.io/docs/reference/data-types#instants
+     * @type {Instant}
+     */
+    insertInstant;
+
+    /**
+     * Details about each application that trust was established for.
+     * @type {StartInstant}
+     */
+    startInstants;
+
+    /**
+     * Additional attributes/state trust.
+     * @type {Map<String, Object>}
+     */
+    state;
+
+    /**
+     * The user's tenant Id.
+     * @type {string}
+     */
+    tenantId;
+
+    /**
+     * The user's Id.
+     * @type {string}
+     */
+    userId;
+}
+```
+
+See [Application API](/docs/apis/applications) documentation for the `context.application` object details.
+
+See [Advanced Threat Detection](/docs/operate/secure/advanced-threat-detection) for more information on this licensed feature.
```

Comment:
```suggestion
For more information about `context.application`, see the [Application API reference](/docs/apis/applications).

For more information on advanced threat protection, see the [Advanced Threat Detection](/docs/operate/secure/advanced-threat-detection) documentation.
```
Swap the ordering of [what] and [where] so users don't have to backtrack to the link.